Source/WebGPU/WGSL/WGSL.cpp

```diff
@@ -126,4 +127,12 @@ PrepareResult prepare(ShaderModule& ast, const String& entryPointName, const std
     return prepareImpl(ast, pipelineLayouts);
 }
 
+ConstantValue evaluate(const AST::Expression& expression, const HashMap<String, ConstantValue>& constants)
+{
+    if (auto constantValue = expression.constantValue())
+        return *constantValue;
+    ASSERT(is<const AST::IdentifierExpression>(expression));
```

Comment:
this assert is redundant, downcast does it already.

Reply:
That is fair. This was mostly as a reminder, since this won't always be true (it's just a limitation of the current implementation that we can't handle the other cases), so when we inevitably crash here we get a better error message.